content/docs/command-reference/params/diff.md

```diff
@@ -1,8 +1,7 @@
 # params diff
 
-Show changes in [parameters](/doc/command-reference/params) between commits in
-the <abbr>DVC repository</abbr>, or between a commit and the
-<abbr>workspace</abbr>.
+Show changes in `dvc params` between commits in the <abbr>DVC repository</abbr>,
```

Comment:
previous was better I think

Reply:
Happy to roll back. The idea here is that the cmd ref should be self-contained so linking between references when possible makes sense. Alternatively we could use the tooltip instead of link.

Reply:
Feel free to commit:
```suggestion
Show changes in <abbr>params</abbr> between commits in the <abbr>DVC repository</abbr>,
```